src/widget-components/mangrove-alerts/config.js

```diff
@@ -133,9 +133,9 @@ export const CONFIG = {
         height: 50,
         // eslint-disable-next-line react/prop-types
         content: ({ payload }) => {
-          const labels = payload.map(({ color, value, payload }) => ({
+          const labels = payload.map(({ color, value, payloadChild }) => ({
```

Comment:
```suggestion
          const labels = payload.map(({ color, value, payload: payloadChild }) => ({
```

Reply:
To avoid this error

![Screenshot 2019-10-15 at 13 56 46](https://user-images.githubusercontent.com/33252015/66830162-1da32300-ef55-11e9-806b-73ad8357b7e7.png)